### DISCUSSION TITLE
Moving Save Files between computers

### DISCUSSION THREAD

**Discussion by Jedith-Freelance** [MikeMaximus/gbm#297] [Q&A] [answered]
### So I would like to preface this discussion with the fact that I have little to no idea what I'm doing, so try not to be harsh.

So I finally decided to find a program to back up my save files. I looked at gamesave manager, but that didn't give me an option to do anything for Epic Games (or at least it didn't look like it did.) So I looked around and found this program and it sounded great! But I've run into a snag that probably everyone here knows how to do, but I'm lost even after looking through the manual multiple times.

I've got 2 computers that I use, a personal home computer and a work computer. On my work computer I have a small game that I want to transfer the save data to my home computer and go back and forth as necessary with a thumb drive. Now I know that cloud saving is a thing, but I don't want to set that up right now.

So I set up this program on both computers and have the game monitored on both, but when I try to move the files from my work computer to my personal one, it doesn't give me the saved file: I boot up the game on the personal computer and it doesn't show any game file there.

So...how does this work on backing up my save files? I look in the winRAR files that are there and I have no idea if I have save files or not. By looking through the manual I can see that I have config files, but nothing else.
Can someone give me some help on how to do this? I'm very lost and I want to understand what the program is doing so I know I've set it up correctly and it's doing its intended function.

**Comment by MikeMaximus** [comment, marked as answer]
Hi Jedith,

A thumb drive isn't the optimal way to use GBM and a bit of a pain, but I'll try to explain some ways you can do it.

### Method 1

**This method needs the thumb drive plugged in before starting GBM and left in while it's running.  If this is out of the question, go to Method 2.**

1.  You must set GBM on **both** computers to save backups to the same location on the thumb drive, start with the PC that has the saved game you want to transfer.  Insert the thumb drive and change GBM's backup location in *File -> Settings -> Files and Folders*,  change the *Backup Folder* to where ever you desire on the thumb drive.  If you receive a message about data already existing in the backup folder,  just choose the recommended option.

2.  Once you have done this on the first PC, Choose the game from the list in GBM and click Backup.  This will run a manual backup and save it to the thumb drive.

3.  Shut down GBM, remove the thumb drive and move to the other PC.  Perform step 1 on the other PC.  Choose the game from the list and click Restore to copy the latest saved game to this PC from the thumb drive.

4.  From this point on all you need to do is move the thumb drive between your PCs and just make sure it's plugged in before GBM starts and left in.  If you'd like GBM to automatically restore new backups when it starts, you can enable that option in *File -> Settings -> Backup and Restore* 

### Method 2

**This method is more work, but doesn't require the thumb drive to be always plugged in.**

1.  On the PC with the latest backups,  browse to the GBM backup folder in File Explorer and copy the 7z (7-Zip File) backup file(s) for each game to your thumb drive.

2.  On the other PC, insert the thumb drive and open GBM.  Choose *Tools -> Import Backup -> Files* from the menu, select the backup file from the thumb drive and click Open.  If you're moving multiple backup files, you can also use the *Folder* option to do it faster.

3.  Choose the game(s) from the list in GBM and click Restore.  You'll need to perform these steps each time you want to move saves between PCs.

I hope this helps you out!

**Reply by Jedith-Freelance** [reply]
So I tried the 2nd method before I had even made this discussion and it didn't seem to work. Now, I might have not done it in exactly the right way, I'll concede. But it seemed close enough that it shouldn't have been an issue. I know it's a bit of a big ask, but is there any nuances to this method that I could have missed?

**Reply by MikeMaximus** [reply]
Hmm if you followed those steps it should have been fine, it's possible the backup configuration for that game isn't doing a backup of the correct location / files then.

**Reply by Jedith-Freelance** [reply]
Now this is a game I've added with the "New Game Wizard", so I don't know how differently that works over one that is in the official game list. Is the software creating its own save files or does it have to know where the "official" ones are that the game uses?

**Reply by MikeMaximus** [reply]
When you manually add a game via the Game Manager or Add Game Wizard you need to know where the save files for the game are located and give it the correct location to backup.  

You can import pre-made configurations from *Files -> Import Games -> Official List* or *Files -> Import Games -> Ludusavi Manifest*

**Reply by Jedith-Freelance** [reply]
Okay, that must be where I went screwy. I was setting that as the location of where Game Manager saves the backups, not the location of the game save files.
Okay, I'll try that today and it should work now.

**Reply by MikeMaximus** [reply]
I hope you got it worked out.
